collect_app/src/main/java/org/odk/collect/android/formlists/savedformlist/SavedFormListListMenuProvider.kt

```diff
@@ -0,0 +1,83 @@
+package org.odk.collect.android.formlists.savedformlist
+
+import android.content.Context
+import android.view.Menu
+import android.view.MenuInflater
+import android.view.MenuItem
+import android.view.MenuItem.OnActionExpandListener
+import androidx.appcompat.widget.SearchView
+import androidx.core.view.MenuProvider
+import org.odk.collect.android.R
+import org.odk.collect.android.formlists.sorting.FormListSortingBottomSheetDialog
+import org.odk.collect.android.formlists.sorting.FormListSortingOption
+
+class SavedFormListListMenuProvider(private val context: Context, private val viewModel: SavedFormListViewModel) : MenuProvider {
+    override fun onCreateMenu(menu: Menu, menuInflater: MenuInflater) {
+        menuInflater.inflate(R.menu.saved_form_list_menu, menu)
+
+        menu.findItem(R.id.menu_filter).apply {
+            setOnActionExpandListener(object : OnActionExpandListener {
+                override fun onMenuItemActionExpand(p0: MenuItem): Boolean {
```

Comment:
Can we call the parameter `menuItem`, not `p0` as we do in `BlankFormListMenuProvider`?